Atomic/syscontainers.py

```diff
@@ -407,13 +407,12 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
                     pass
             rootfs = os.path.join(destination, "rootfs")
 
-        if os.path.exists(destination):
-            shutil.rmtree(destination)
-
```

Comment:
Should we keep this rmtree  logic but hide it behind an if not upgrade?

if os.path.exists(destination) and not upgrade:
    shutil.rmtree(destination)